src/test/components/TrackThread.test.js

```diff
@@ -177,25 +174,4 @@ describe('timeline/TrackThread', function() {
       selectionEnd: 2,
     });
   });
-
-  it('changes the selected thread when clicking a marker', function() {
-    const profile = getMarkersProfile();
-    profile.threads.push(getEmptyThread());
-    const { getState, dispatch, tracingMarkersCanvas } = setup(profile);
-    const thisThread = 0;
-    const otherThread = 1;
-
-    dispatch(changeSelectedThread(otherThread));
-    expect(getSelectedThreadIndex(getState())).toBe(otherThread);
-
-    tracingMarkersCanvas.simulate(
-      'mousedown',
-      getMouseEvent({ pageX: STACK_1_X_POSITION })
-    );
-    tracingMarkersCanvas.simulate(
-      'mouseup',
-      getMouseEvent({ pageX: STACK_1_X_POSITION })
-    );
-    expect(getSelectedThreadIndex(getState())).toBe(thisThread);
-  });
```

Comment:
I decided to remove the test altogether as I assume that clicking in a track is tested elsewhere.

Reply:
Works for me.